client/my-sites/hosting/github/github-connect-card/use-github-branches-query.ts

```diff
@@ -32,7 +32,6 @@ export const useGithubBranchesQuery = (
 			meta: {
 				persist: false,
 			},
-			refetchOnWindowFocus: false,
```

Comment:
The reason we didn't refetch on focus is that if the user switches tabs, they'll lose their current selection 😬

Try it:

1. select a branch that's not the default
2. switch tabs
3. check that the selection is gone and you're back to the default branch

Reply:
I'm the changing the logic which auto selects the branch to check for an existing selection.

Reply:
> It would be nice to find a less redundant message than the one below, though.

I think we can just leave the text field blank in this case.

A (!) red error is a bit more than I'd expect in this case. There's been no user error they should fix up, they haven't typed anything wrong. It feels more like an informational thing.

But I am definitely in favour of showing a more accessible message!